packages/voting/migrations/sqls/20211029155918-elections-group-slug-down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "elections"
+  DROP COLUMN "groupSlug"
+;
```

Comment:
(Missing new line at end of file.)

```suggestion
ALTER TABLE "elections"
  DROP COLUMN "groupSlug"
;

```